mission-racing/src/main/java/com/racing/exception/CarNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.racing.exception;
+
+public class CarNotFoundException extends RuntimeException{
```

Comment:
CustomException 좋습니다!
정말 중요한 개념을 잡고 갔으면 좋겠어서 몇 가지 질문 남겨봅니다!

1. `Checked Exception과 Unchecked Exception의 차이는 무엇일까요?`
2. 해당 CustomException에서 `RuntimeException`을 상속받은 이유는 무엇인가요?
3. `CarNotFoundException`예외가 비즈니스 로직에서 발생하였을 때 클라이언트 측에게 응답은 어떻게 갈까요? 이 응답을 404 NOT_FOUND 로 주고 싶다면 어떻게 해야할까요?

```suggestion
public class CarNotFoundException extends RuntimeException {
```

Reply:
### Custom Exception

1. checked Exception
- 컴파일 시점에 컴파일러에서 확인하는 예외
- 예외 처리를 반드시 해줘야 한다

2. unchecked Exception
- 런타임에 발생한다.
- 프로그램 논리 내부에 오류가 있을때 발생

비즈니스 로직에서 발생하는 예외는 예외 처리를 강제할 필요가 없기 때문에 unchecked Exception으로 처리한다.

3. @ExceptionHandler를 사용하여 예외 응답을 따로 설정하여 전달할 수 있을 것 같습니다.!